e2e/imgbuild/imgbuild.go

```diff
@@ -163,7 +673,11 @@ func RunE2ETests(t *testing.T) {
 	// builds from definition file and URI
 	t.Run("From", buildFrom)
 	// build and image from an existing image
-	t.Run("multistage", buildMultiStage)
+	t.Run("multistage", buildMultiple)
```

Comment:
I'd prefer not naming these similarly. `multistage` and `MultiStage` below is a bit confusing.